corehq/apps/domain/views.py

```diff
@@ -701,20 +701,19 @@ def get_product_summary(self, plan_version, account, subscription):
         product_info = {
             'name': product_rate.product.product_type,
             'monthly_fee': _("USD %s /month") % product_rate.monthly_fee,
-            'credit': None,
             'type': product_rate.product.product_type,
         }
-        credit_lines = None
-        if subscription is not None:
-            credit_lines = CreditLine.get_credits_by_subscription_and_features(
-                subscription, product_type=product_rate.product.product_type
-            )
-        elif account is not None:
-            credit_lines = CreditLine.get_credits_for_account(
-                account, product_type=product_rate.product.product_type
-            )
-        if credit_lines:
-            product_info['credit'] = self._fmt_credit(self._credit_grand_total(credit_lines))
+
+        total_product_credit = CreditLine.get_total_credits(
+            subscription=subscription,
+            account=account,
+            product_type=product_rate.product.product_type
+        )
+        if total_product_credit is not None:
+            product_info['credit'] = self._fmt_credit(total_product_credit)
+        else:
+            product_info['credit'] = None
```

Comment:
Is there a reason you are returning `None` here, instead of whatever `_fmt_credit` returns?